addon-test-support/index.js

```diff
@@ -1,2 +1,9 @@
 import setupMirage from './setup-mirage';
 export { setupMirage };
+
+import { dependencySatisfies } from '@embroider/macros';
+
+if (dependencySatisfies('ember-qunit', '*')) {
+  window.QUnit.config.urlConfig.push({ id: 'mirageLogging', label: 'Mirage logging', });
```

Comment:
are you sure that this works with ember-qunit v5? I thought we now had to explicitly import it and it was no longer a global on `window`.

/cc @rwjblue

Reply:
Originally the code was in ember-cli-build.js, when I looked there QUNIT was no longer a global. Thats where the need to import came from. To fix the problem I moved the code from ember-cli-build.js to test-support/index.js. After more testing it turns out it is a global again there (I would assume making qunit a peer-dependancy caused a load order problem and it was no longer a global in ember-cli-build.js but was when test-support/index.js got around to executing). So I could have just gone with the original code

Reply:
I have tested with by temporarily upgrading the tests to ember-qunit v5 and it does work.  I did not leave it that way because I would also need to upgrade the test projects because of the way they share the node-modules directory.